com.supermodel3.Supermodel.yaml

```diff
@@ -0,0 +1,38 @@
+id: com.supermodel3.Supermodel
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: supermodel
+separate-locales: false
+finish-args:
+  - --device=all
+  - --persist=.supermodel
+  - --filesystem=host:ro
```

Comment:
this is a very broad filesystem permission

Reply:
Emulator reads external rom files, uses various joystick devices

Reply:
host doesn't do permissions for joysticks. Also `home` overlaps with `host`. Still remain excessively broad.

Reply:
device=all <-- To allow for all sorts of gamepads, joysticks, wheels, etc.
persist=.supermodel <-- Default home directory
filesystem=home <-- Seems to require this in order for the app to create its own folders and files for save states, NVRAM, etc.
filesystem=host:ro <-- To allow reading ROM files from disparate filesystems, mounted media, etc.
share=network <-- Application allows to link up multiple arcade machines
share=ipc <-- For X11
socket=fallback-x11 <-- For X11
socket=pulseaudio <-- PulseAudio required
socket=wayland <-- Wayland supported
allow=bluetooth <-- To support bluetooth gamepad, controllers, etc.

Reply:
give that there is `.persist` why does it need to write in home?